conf/src/main/java/org/apache/nemo/conf/JobConf.java

```diff
@@ -332,6 +316,31 @@
   public final class ExecutorId implements Name<String> {
   }
 
+  /**
+   * Maximum off-heap memory ratio to the total memory in the executor.
+   */
+  @NamedParameter(doc = "The maximum ratio of off-heap memory size to the total memory size.",
+    short_name = "max_offheap_ratio", default_value = "0.2")
+  public final class MaxOffheapRatio implements Name<Double> {
+  }
+
+  /**
+   * Maximum off-heap memory size in the executor.
+   * This is set by the system according to the off-heap ratio.
+   */
+  //@NamedParameter(doc = "The maximum off-heap memory that can be allocated")
+  //public final class MaxOffheapMb implements Name<Integer> {
+  //}
```

Comment:
Let's get rid of the unused configuration

Reply:
I've deleted this configuration and changed the order of other configurations in an appropriate way, also added short name for ExecutorMemoryMb as "memory_mb", as we look for memory in runtime master (function request container) with this string.